demo/webpack/README.md

```diff
@@ -1,6 +1,9 @@
 ## rsk3.js webpack demo
 
-1. `npm run build` to generate `./dist/main.js`.
-2. open `./src/index.html` to view page.
+1. Run `npm install`.
 
-You may update the code in `./src/index.js`, and repeat the above steps.
+2. Run `npm run build` to generate `./dist/main.js`.
+
+3. open `./src/index.html` to view the page in your browser.
```

Comment:
maybe add `in Chrome or Firefox` to be clear.

open `./src/index.html` in Chrome or Firefox